src/Models/ConnectedAccount.php

```diff
@@ -32,6 +33,17 @@ class ConnectedAccount extends Model
         'expires_at',
     ];
 
+    public function __construct(array $attributes = [])
+    {
+        $config = Config::get('socialment');
```

Comment:
May I suggest the idea from my [PR](https://github.com/chrisreedio/socialment/pull/65) here?
To use helpers instead of facades (explanation in the discussion)

In this commit, we have two different coding styles: the config facade + helper combination, which seems a bit overengineered

Since you're not opposed to using helpers (as discussed in the method below), I suggest we apply the same approach here, above, and use it as well

Reply:
@chrisreedio Don’t mind me, I’m just watching your every move... in the most non-creepy way possible! :grin: